wpilibj/src/shared/java/edu/wpi/first/wpilibj/command/InstantCommand.java

```diff
@@ -0,0 +1,22 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2016. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.command;
+
+public class InstantCommand extends Command {
+  public InstantCommand() {
+    super();
```

Comment:
I'd forgotten that... I successfully tested removing the "super();" line, so you can go ahead and do that.